fast64_internal/f3d/f3d_gbi.py

```diff
@@ -1984,75 +1985,8 @@ def to_binary(self):
         )
 
     def to_c(self):
-        if bpy.context.scene.decomp_compatible:
-            return (
-                "{{"
-                + ", ".join(
-                    (
-                        (
-                            "{"
-                            + ", ".join(
-                                (
-                                    str(self.position[0]),
-                                    str(self.position[1]),
-                                    str(self.position[2]),
-                                )
-                            )
-                            + "}"
-                        ),
-                        "0",
-                        "{" + str(self.uv[0]) + ", " + str(self.uv[1]) + "}",
-                        (
-                            "{"
-                            + ", ".join(
-                                (
-                                    "0x" + format(self.colorOrNormal[0], "X"),
-                                    "0x" + format(self.colorOrNormal[1], "X"),
-                                    "0x" + format(self.colorOrNormal[2], "X"),
-                                    "0x" + format(self.colorOrNormal[3], "X"),
-                                )
-                            )
-                            + "}"
-                        ),
-                    )
-                )
-                + "}}"
-            )
-        else:
-            return (
-                "{"
-                + ", ".join(
-                    (
-                        str(self.position[0]),
-                        str(self.position[1]),
-                        str(self.position[2]),
-                        "0",
-                        str(self.uv[0]),
-                        str(self.uv[1]),
-                        "0x" + format(self.colorOrNormal[0], "X"),
-                        "0x" + format(self.colorOrNormal[1], "X"),
-                        "0x" + format(self.colorOrNormal[2], "X"),
-                        "0x" + format(self.colorOrNormal[3], "X"),
-                    )
-                )
-                + "}"
-            )
-
-    def to_sm64_decomp_s(self):
-        return "vertex " + ", ".join(
-            (
-                str(self.position[0]),
-                str(self.position[1]),
-                str(self.position[2]),
-                "0",
-                "0x" + format(self.uv[0], "X"),
-                "0x" + format(self.uv[1], "X"),
-                "0x" + format(self.colorOrNormal[0], "X"),
-                "0x" + format(self.colorOrNormal[1], "X"),
-                "0x" + format(self.colorOrNormal[2], "X"),
-                "0x" + format(self.colorOrNormal[3], "X"),
-            )
-        )
+        spc = lambda x: ", ".join([str(a) for a in x])
```

Comment:
this applies to other lambda usage
PEP8 https://peps.python.org/pep-0008/ :
> Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier:

Reply:
In that case, I should change the defined lambdas to be like this then
```suggestion
    def size_total(self, f3d):
        def use_siz_tot(x):
            return isinstance(command, SPDisplayList) and command.displayList.DLFormat != DLFormat.Static
```
```suggestion
    def to_c(self):
        def spc(x):
            return ", ".join([str(a) for a in x])
```

Reply:
yeah thats the idea